app/src/main/res/raw/okta_app_auth_config.json

```diff
@@ -1,6 +1,7 @@
 {
   "client_id": "{your-client-id-here}",
   "redirect_uri": "{your-redirect-uri}",
+  "end_session_redirect_uri": "{your-redirect-uri}",
```

Comment:
@nbarbettini we may want to add these `redirect_uri` to our config checks

Reply:
Ideally, the mock values here match what is shown [in our README](https://github.com/okta/okta-sdk-appauth-android/pull/38/files#diff-04c6e90faac2675aa89e2176d2eec7d8R76). Can we please update these?